### PR TITLE
playground input updates

### PR DESCRIPTION
**Pull request by thompsongl**
Accounting for differences in docgenInfo output and playground component expectations

cc: @anishagg17 for visibility. Let us know if you see any other differences that I missed.

### CODE REVIEW

File: src-docs/src/services/playground/props.js [hunk 1]
```diff
@@ -23,10 +23,7 @@ const getProp = (prop, propName) => {
       newProp.required = prop.required;
       if (prop.defaultValue) {
         newProp.defaultValue = prop.defaultValue.value;
-        newProp.value = prop.defaultValue.value.substring(
-          1,
-          prop.defaultValue.value.length - 1
-        );
+        newProp.value = prop.defaultValue.value;
```
Comment (anishagg17) [inline src-docs/src/services/playground/props.js:26]: Screenshot 2020-07-14 at 12 01 18 AM

Some times `defaultValue` is `null`

so, 
```
        newProp.value = prop.defaultValue && prop.defaultValue.value;
```

**Review by anishagg17 — APPROVED** [review]
Pulled and checked , working properly.

**Comment by thompsongl** [comment]
> Pulled and checked , working properly.

Thanks for checking 👍

File: src-docs/src/services/playground/props.js (hunk 1 above)
Comment (thompsongl) [inline src-docs/src/services/playground/props.js:26]:
```suggestion
        newProp.value = prop.defaultValue && prop.defaultValue.value;
```

File: src-docs/src/services/playground/props.js [hunk 2]
```diff
@@ -23,10 +23,7 @@ const getProp = (prop, propName) => {
       newProp.required = prop.required;
       if (prop.defaultValue) {
         newProp.defaultValue = prop.defaultValue.value;
-        newProp.value = prop.defaultValue.value.substring(
-          1,
-          prop.defaultValue.value.length - 1
-        );
+        newProp.value = prop.defaultValue && prop.defaultValue.value;
```
Comment (anishagg17) [inline src-docs/src/services/playground/props.js]: NO need of this ` prop.defaultValue` because of `if else` on line 24

Comment (thompsongl) [inline src-docs/src/services/playground/props.js]:
```suggestion
        newProp.value = prop.defaultValue.value;
```

**Pull request merged**